mithril-infra/assets/docker/docker-compose-monitoring.yaml

```diff
@@ -52,6 +52,44 @@ services:
       options:
         tag: "{{.ImageName}}|{{.Name}}|{{.ImageFullID}}|{{.FullID}}"
 
+  process-exporter:
+    image: ncabatoff/process-exporter:latest
+    container_name: process-exporter
+    profiles:
+      - tools
+      - monitoring
+      - all
+    restart: unless-stopped
```

Comment:
Why aren't the two processes configured the same way?

```
process-exporter:
  restart: unless-stopped
```

```
disk-usage-exporter:
    restart: always
```

Reply:
I switched it to `always`